tools/testsys/src/aws_resources.rs

```diff
@@ -493,6 +496,7 @@ impl AwsEcs {
             },
             spec: ResourceSpec {
                 depends_on: None,
+                conflicts_with: None,
```

Comment:
I'd like for `ResourceSpec` to implement the `Default` trait so we can populate these `None` fields automatically via `..Default::default()`.